lib/internal/test_runner/mock/mock.js

```diff
@@ -45,19 +45,36 @@ class MockFunctionContext {
     this.#times = times;
   }
 
+  /**
+   * Returns an array of all function calls.
```

Comment:
I think this is too generic. MAYBE:

```js
  /**
   * Gets an array of recorded calls made to the mock function.
   *
   * @returns {Array} An array of recorded calls.
   */
```

Reply:
Agreed, will be more specific for the next time, thank you!